docs/releases/1.11.1.txt

```diff
@@ -21,3 +21,6 @@ Bugfixes
 
 * Fixed ``QuerySet.filter()`` crash when it references the name of a
   ``OneToOneField`` primary key (:ticket:`28047`).
+
+* Fixed ``MultiValueDictError`` double-quoting missing keys in error messages
```

Comment:
You can remove the release note as this won't be backported per our [supported versions policy](https://docs.djangoproject.com/en/dev/internals/release-process/#supported-versions).